### PR TITLE
Bump VMWARE_CONTAINER_VERSION to v2.0.0-4

### PR DESCRIPTION
**Pull request by pkliczewski**
Signed-off-by: Piotr Kliczewski <piotr.kliczewski@gmail.com>

```release-note
Bump VMWARE_CONTAINER_VERSION to v2.0.0-4 
```

### CODE REVIEW

**Comment by pkliczewski** [comment]
@tiraboschi FYI

**Comment by ovirt-infra** [comment]
All tests passed

File: deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -1445,7 +1445,7 @@ spec:
                 - name: CONVERSION_CONTAINER
                   value: quay.io/kubevirt/kubevirt-v2v-conversion:v2.0.0
                 - name: VMWARE_CONTAINER
-                  value: quay.io/kubevirt/kubevirt-vmware:v2.0.0-3
+                  value: quay.io/kubevirt/kubevirt-vmware:v2.0.0-4
```
Comment (tiraboschi) [inline deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml:1448]: latest versions of the CSV on master is 1.2.0 (1.1.0 is the current one on release-2.4 branch). Did you run hack/build-manifests.sh after updating hack/config ?

Comment (pkliczewski) [reply deploy/olm-catalog/kubevirt-hyperconverged/1.1.0/kubevirt-hyperconverged-operator.v1.1.0.clusterserviceversion.yaml:1448]: no, will do

**Comment by ovirt-infra** [comment]
All tests passed

**Comment by orenc1** [comment]
i think we forgot to backport this to 2.4
/cherry-pick release-2.4

**Pull request merged**